distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -1176,29 +1174,29 @@ public void testValidationEnforced() throws Exception {
                     "/GetRecordsResponse/SearchResults/Record[identifier=\"%s\"]",
                     id2))));
 
-            deleteMetacard(id1);
         } finally {
+            deleteMetacard(id1);
             getServiceManager().stopFeature(true, "catalog-plugin-metacard-validation");
         }
     }
 
     @Test
     public void testValidationUnenforced() throws Exception {
         getServiceManager().startFeature(true, "catalog-plugin-metacard-validation");
+        Configuration metacardMarker = configAdmin.getConfiguration(
+                "ddf.catalog.metacard.validation.MetacardValidityMarkerPlugin",
+                null);
+        Dictionary<String, Object> properties = new Hashtable<>();
+        List<String> enforcedMetacardValidators = new ArrayList<>();
+        enforcedMetacardValidators.add("");
+        properties.put("enforcedMetacardValidators", enforcedMetacardValidators);
+        metacardMarker.update(properties);
+
+        String id1 = ingestXmlFromResource("/metacard1.xml");
+        String id2 = ingestXmlFromResource("/metacard2.xml");
 
         try {
             // Update metacardMarkerPlugin config with no enforcedMetacardValidators
-            Configuration metacardMarker = configAdmin.getConfiguration(
-                    "ddf.catalog.metacard.validation.MetacardValidityMarkerPlugin",
-                    null);
-            Dictionary<String, Object> properties = new Hashtable<>();
-            List<String> enforcedMetacardValidators = new ArrayList<>();
-            enforcedMetacardValidators.add("");
-            properties.put("enforcedMetacardValidators", enforcedMetacardValidators);
-            metacardMarker.update(properties);
-
-            String id1 = ingestXmlFromResource("/metacard1.xml");
-            String id2 = ingestXmlFromResource("/metacard2.xml");
 
             // Search for all entries, implicit "validation-warnings is null" and "validation-errors is null"
             // should get added by MetacardValidityCheckerPlugin
```

Comment:
Good catch, but I think saying that it should get added by `ValidationQueryFactory` would actually be the most accurate.